.github/workflows/build.yml

```diff
@@ -54,6 +54,18 @@ jobs:
       - name: Checkout Push to Registry action
         uses: actions/checkout@v4
 
+      # Confirm that cosign.pub matches SIGNING_SECRET
+      - uses: sigstore/cosign-installer@v3.2.0
+
+      - name: Check SIGNING_SECRET matches cosign.pub
+        shell: bash
+        run: |
+          diff -u <(cosign public-key --key env://COSIGN_PRIVATE_KEY) cosign.pub
```

Comment:
Does `diff -u` cause the build to fail if they don't match?

Reply:
Yep. I tried it out on my personal fork by intentionally making the public key mismatch. Here's how the failure looks:

![image](https://github.com/ublue-os/startingpoint/assets/14993/e0d6ab35-50ab-42a6-8e55-8865b62e97c8)

Reply:
I tried to make the title of the action (`Check SIGNING_SECRET matches cosign.pub`) explain what's going on but I'll see if I can make it clearer.